app/packs/src/components/container/ContainerDatasetModalContent.js

```diff
@@ -523,7 +518,7 @@ export default class ContainerDatasetModalContent extends Component {
               )}
               {annotateButton(attachment, this)}
               {moveBackButton(attachment, this.handleAttachmentBackToInbox, readOnly)}
-              &nbsp;
+                <span className='me-2'>{''}</span>
```

Comment:
This seems even more wrong than a &nbsp;.
Nest one of the soon to be released ButtonContainer Components with the tightly grouped buttons and throw that into a second ButtonContainer together with the single Button.